instrumentation/grpc/src/main/java/brave/grpc/TracingServerInterceptor.java

```diff
@@ -136,6 +136,10 @@ public void sendMessage(RespT message) {
       SpanInScope scope = tracer.withSpanInScope(span);
       try { // retrolambda can't resolve this try/finally
         delegate().onHalfClose();
+      } catch (RuntimeException | Error e) {
```

Comment:
is the intent to close the span on half-close only on error? maybe add a comment why

Reply:
I have replicated the behavior of the 'close' method, but I think there would be no problem in capturing Throwable instead.

Reply:
I guess then if half-close occurs then close wouldn't? sorry I'm not aware of the state diagram.

Reply:
So, I suppose a comment would be something like..

```
// if there was an exception executing onHalfClose, we don't expect other lifecycle commands
// Accordingly, we close the span
```

Reply:
```suggestion
      } catch (RuntimeException | Error e) {
      // if there was an exception executing onHalfClose, we don't expect other lifecycle commands
      // Accordingly, we close the span
```

Reply:
I will edit the file directly